core/src/main/java/com/graphhopper/routing/util/parsers/OSMMaxAxleLoadParser.java

```diff
@@ -32,15 +32,9 @@
 public class OSMMaxAxleLoadParser implements TagParser {
 
     private final DecimalEncodedValue maxAxleLoadEncoder;
-    private final boolean enableLog;
 
     public OSMMaxAxleLoadParser() {
-        this(MaxAxleLoad.create(), false);
-    }
-
-    public OSMMaxAxleLoadParser(DecimalEncodedValue maxAxleLoadEncoder, boolean enableLog) {
-        this.maxAxleLoadEncoder = maxAxleLoadEncoder;
-        this.enableLog = enableLog;
+        this.maxAxleLoadEncoder = MaxAxleLoad.create();
```

Comment:
I would not remove this. I think the additional constructor is a meaningful addition, so that the user could increase or decrease the precision. (same for the other parsers)